OrbitCore/LinuxEventTracer.h

```diff
@@ -0,0 +1,26 @@
+//-----------------------------------
+// Copyright Pierric Gimmig 2013-2019
+//-----------------------------------
+#pragma once
+
+#include <thread>
+
+//-----------------------------------------------------------------------------
+class LinuxEventTracer
+{
+public:
+    typedef std::function<void(const std::string& a_Data)> Callback;
+    void Start();
+    void Stop();
+    bool IsRunning() const { return !m_ExitRequested; }
+
+protected:
+    void CommandCallback(std::string a_Line);
+
+private:
+    bool m_ExitRequested = true;
```

Comment:
Seems counterintuitive that this is set to `true` by default, that is, "exit has been requested". Maybe the name is not chosen well? The meaning of this variable seems to be, the tracer is running.

Reply:
Well, I agree that the default value with true, is not intuitive.
However, the main usage of that variable is in the while(!m_ExitRequested) that basically runs until the User presses "X" in the CaptureView in order to stop tracing. In particular, we would not stop until all the remaining events has been processed. So it is more about the Request from the user, as about the current state (i.e. whether it is running or not).

Basically, I copied the name from the Bpftrace implementation. However, I am happy to change it, to something more meaningful.